tcf_website/static/css/site/components/stats_summary.css

```
@@ -0,0 +1,58 @@
+  .stats-summary .stats-card__value {
+    font-size: var(--text-3xl);
+  }
+}
+
+.stats-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+
+  padding: var(--space-5) var(--space-4);
```

Stylelint (17.6.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)